### PR TITLE
Particle and sound classes

### PR DESCRIPTION
**Pull request by YanisBft**

### CODE REVIEW

**Review by shartte — APPROVED** [review]

File: mappings/net/minecraft/particle/ParticleType.mapping [hunk 1]
```diff
@@ -1,8 +1,9 @@
 CLASS net/minecraft/class_2396 net/minecraft/particle/ParticleType
-	FIELD field_11196 shouldAlwaysSpawn Z
+	FIELD field_11196 alwaysShow Z
```
Comment (shartte) [inline mappings/net/minecraft/particle/ParticleType.mapping:2]: IIRC didnt this mean that the particle type should be shown _to other players_ other than the "current" one? Or was I mistaken here.

Comment (YanisBft) [reply mappings/net/minecraft/particle/ParticleType.mapping:2]: No it's used to still display particles if you have particles disabled (attack indicator, campfire, sweep attack and explosions for instance)

Comment (liach) [reply mappings/net/minecraft/particle/ParticleType.mapping:2]: `required`?

Comment (YanisBft) [reply mappings/net/minecraft/particle/ParticleType.mapping:2]: added a line of javadoc to make the param clearer

**Review by liach — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Pull request merged**